SNIPS/snip-75.md

```diff
@@ -0,0 +1,213 @@
+| Vetted time delayed       | Someone (currently, only SW or the council) | 50%       | Regular         |
+| Non-vetted, no time delay | Security council                     | 66%       | None            |
+
+**Note**: Regular time delay SHOULD be set to at least 7 days to be in line with l2beat guidelines.
+
+### Vetting Failure Report
+In the event that the evaluation of an upgrade does not receive the requisite number of votes (50% for a regular action and 66% for an emergency action), the Security Council SHALL produce a "Vetting Failure Report" within fourteen days of the vote on the upgrade, including:
+- The rationale behind the negative votes cast by members.
+- The rationale of the members who did approve the upgrade, if any.
+- An enumeration of the issues identified with the upgrade.
+- Potential avenues for resolving said issues, if feasible.
+
+The Security Council's sole grounds for either rejecting or approving upgrades will be security concerns.
+
+## Security Concerns
+The security council shall be responsible for assessing the extent to which a security concern is present. Given the nature of such concerns, it is not possible to produce a comprehensive list, so the following list is non-exhaustive and provided for illustrative purposes only:
+- **Smart Contract Vulnerabilities**: Bugs, exploits, and vulnerabilities that can lead to theft of funds, manipulation of the network, or other unintended consequences.
+- **Attacks and Hacks**: Sybil, Denial of Service (DoS), Double-Spending, Phishing, Social Engineering.
+- **Stability and Integrity Issues**
+- **Liveness Issues**
+- **Malicious or Rogue Nodes**
+- **Oracle Manipulation**
+
+### Security Emergency
+The Security Council is permitted to preemptively address actual or anticipated bugs, defects, unplanned maintenance, stability issues, integrity, availability, non-repudiation, or other security issues with Starknet. 
+
+These Emergency response measures MAY be taken without specific Governance approval.
+
+If the Security Council ever utilizes this discretion, it MUST provide the community with a prompt and comprehensive retrospective (within the bounds of any legal commitments to or security requirements for confidentiality) after the action is taken, explaining the rationale for it.
+
+### Eligibility
+Members MUST meet the following criteria:
+- **Technical Competency**: Familiarity with Starknet and secure key management.
+- **Reputation and Alignment**: Trustworthy individuals aligned with Starknet’s vision and values.
+- **Diversity**: Geographic and organizational diversity to prevent overconcentration, with less than 50% from any one country, and fewer than four people from the same organization.
+- **Screening**: All members must pass KYC/AML checks and sign a standard contract accepting their responsibilities within the role.
+
+### Compensation
+Council members receive a monthly stipend paid in STRK tokens for their contributions.
```

Comment:
Where is this amount decided? Based on what? Is it public?

Reply:
An amount was decided it should be public indeed I just didn't feel like it belonged to the PR 😅